sqlmesh/core/plan/evaluator.py

```diff
@@ -480,7 +492,10 @@ def evaluate(
                 self.dag_run_poll_interval_secs,
             )
             if not plan_application_succeeded:
-                raise PlanError("Plan application failed.")
+                msg = "Error: Plan application failed."
+                self.console.log_error(msg)
+                logger.info(msg)
+                raise PlanApplyError
```

Comment:
[Ditto] Let's not raise exceptions without messages. Also why can't this be handled by the click's exception handler